tslint.json

```diff
@@ -0,0 +1,100 @@
+{
+    "defaultSeverity": "error",
+    "extends": "typestrict",
+    "jsRules": {},
+    "rules": {
+        "arrow-parens": true,
+        "ban-ts-ignore": true,
+        "import-spacing": true,
+        "indent": {
+            "options": [
+                "spaces",
+                2
+            ]
+        },
+        "max-line-length": {
+            "options": {
+                "limit": 120
+            }
+        },
+        "no-debugger": true,
+        "no-console": true,
+        "new-parens": true,
+        "only-arrow-functions": {
+            "options": [
+                "allow-declarations",
+                "allow-named-functions"
+            ]
+        },
+        "no-consecutive-blank-lines": [
+            true,
+            2
+        ],
+        "no-irregular-whitespace": true,
+        "no-trailing-whitespace": {
```

Comment:
There's no reason to have trailing whitespace on lines except within template strings.